doc/user_guide/index.rst

```diff
@@ -47,15 +47,26 @@ PSyclone User Guide
 
 .. only:: html
 
-    PSyclone is a code generation system developed to support domain-specific
-    languages (DSLs) for finite element, finite volume and finite difference
-    codes. Notably, it is used in the
-    `LFRic Project <https://www.metoffice.gov.uk/research/modelling-systems/
-    lfric/>`_, and it also supports the
+    PSyclone is a source-to-source Fortran compiler designed to programmatically
+    optimise and parallelise HPC applications using user-provided transformation
```

Comment:
"optimise, parallelise and instrument HPC..." and s/using/via/

Reply:
Done